JSDemos/Demos/Charts/Candlestick/description.md

```diff
@@ -1 +1,10 @@
-In&nbsp;this demo, the **candle stick** series displays variations in&nbsp;stock price throughout the course of&nbsp;a&nbsp;given day. Each point consists of&nbsp;a&nbsp;rectangle (body) whose bottom and topmost values correspond to&nbsp;the opening and closing price of&nbsp;a&nbsp;stock. A&nbsp;vertical line (shadow, wick or&nbsp;tail) displays the high and low price for the stock. If&nbsp;the stock closes higher than its opening price, no&nbsp;color fills its body and if&nbsp;the stock closes lower than its opening price, a&nbsp;color fill is&nbsp;applied.
\ No newline at end of file
+A Candlestick [series type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day. You can specify the Candlestick series type for each object in the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) or specify the type in the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object for all series in the Chart. Use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/).[candlestick](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/CandleStickSeries/) object to configure properties of the Chart Candlestick series.
```

Comment:
I would change the first sentence to better convey that *a single point* in a Candlestick chart display data over the course of a day. Right now, it seems like the entire chart only displays data for one day. 

The rest of the text then can be a separate paragraph. 

```suggestion
A single point on a Candlestick [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day. 

You can use the Candlestick type for an individual [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) or specify the type in the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object for all series in the Chart. Use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/).[candlestick](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/CandleStickSeries/) object to configure properties of the Chart Candlestick series.
```